src/popup/router/components/BalanceInfo.vue

```diff
@@ -3,43 +3,20 @@
     class="balance-info"
     data-cy="balance-info"
   >
-    <div class="balance-wrapper">
-      <AeBalance :balance="Number(balances[idx])" />
```

Comment:
we did not incorporate currency change settings.

Reply:
They should not be part of a card anymore.